cmd/appender/main.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/google/go-containerregistry/authn"
+	"github.com/google/go-containerregistry/name"
+	"github.com/google/go-containerregistry/v1"
+	"github.com/google/go-containerregistry/v1/remote"
+	"github.com/google/go-containerregistry/v1/tarball"
+	"github.com/google/go-containerregistry/v1/writable"
+)
+
+var (
+	inputFile  = flag.String("tarball", "", "the tarball to append")
+	srcTag     = flag.String("src-tag", "", "The tag of the remote image to append to")
```

Comment:
nit: we should support digests.  The Python version of this calls this `--src-image` and `--dst-image`, see: https://github.com/google/containerregistry/blob/master/tools/docker_appender_.py

Reply:
Let's make that a separate issues & PR

Reply:
Issue: https://github.com/google/go-containerregistry/issues/66

Update diff contains name changes to `src-image` and `dst-image`